[DOCU-1165] Application Overview for Konnect Dev Portal (#3082)
Kong/docs.konghq.com: app/konnect/dev-portal/developers/dev-gen-creds.md

toc: true
---

A credential, or API key, identifies and authenticates the developer application making a request. Use the API key either in the request URL as a query parameter, or in the request header. 

> Also, any credential can be permanently deleted. See [Delete a credential](#delete-a-credential). 

Reviewer: ```suggestion
You can permanently delete any credential at any time. See [Delete a credential](#delete-a-credential). 
```
I suggest changing this into an action and not a passive phrase, because when I see the phrase "any credential can be permanently deleted", I start wondering about who would delete my credential, or why, or any number of other worries.

## Generate a credential

A credential, or API key, generated in the {{site.konnect_short_name}} Dev Portal is a 32-character string associated with an Application. An Application can have multiple credentials. 

1. In the Dev Portal, click **My Apps** from the dropdown menu under your login email.

2. Click the Application for which you want to generate a credential.

3. In the **Authentication** pane, click **Generate Credential**.

4. Test the generated credential by making a call to the service the
   [Application is registered with](/konnect/dev-portal/developers/dev-reg-app-service)

>    using your `key-auth` credential.

Reviewer: ```suggestion
   using your `key-auth` credential:
```

   ```
   <proxy-url>/<service>/?apikey=<apikey>
   ```

## Delete a credential

> A credential can be permanently deleted. Note that a credential cannot be restored. 

Reviewer: ```suggestion
You can permanently delete a credential. Note that the credential cannot be restored. 
```
(meaning the specific, deleted credential, rather than any random credential)

> 1. Navigate to the application details page for an application.

Reviewer: ```suggestion
1. Navigate to an application's details page.
```
This page name doesn't exist, so I don't think we need to worry about literals like "application details page for an application".
